cmd/service.go

```diff
@@ -34,7 +34,7 @@ var serviceCreateCmd = &cobra.Command{
 
 If service name is not provided, service type value will be used.
 
-A full list of service types that can be deployed is available using: 'odo service catalog'`,
+A full list of service types that can be deployed are available using: 'odo catalog list services'`,
```

Comment:
Do you also envision to list the `plans`, `brokers`, `instances` and `bindings` created ?

`odo catalog list services|plans|brokers|bindings`

Reply:
1) Plan detail

You need a command to show the detail of a plan as we can do with `svcat`
We could then use by example : `odo catalog plan <serviceName>/<planName>`

2) Create service
I agree with you for the command to create the service excepted that you must also pass the class name

`odo service create service_name --class class_name --plan plan_name -p parameters`

Remark : As some parameters are json typed, then it will be very difficult to pass them using a one line command and -p, -p ... This is why, we have in our prototype, defined such parameters/info into a yaml config file

3) Mount the secret to component/

To do the binding, then you must also pass the name of the secret

odo link component_name --service service_name --secret secret_name

component_name corresponds to the DeploymentConfig's name
secret_name : secret containing the parameters

Remark : There is a step that you don't convert which is "generating the secret with the parameters of the plan selected. Is it something that you will do automatically during the creation of the service. IF this is what you plan to do, then you must also pass the name of the secret ->

`odo service create service_name --class class_name --plan plan_name --secret secret_name -p parameters`